src/components/stepper/index.js

```diff
@@ -33,7 +33,7 @@ const Stepper = ({ header, children, onComplete }) => {
 
             <div className="stepper-buttons">
                 <Button text="Back" classes="offwhite" onClick={onBackClick} />
-                <Button text={currentStep === children.length-1 ? 'Submit' : 'Next'} classes="blue" onClick={onNextClick} />
+                <Button text={currentStep === children.length-1 ? 'Create profile' : 'Next'} classes="blue" onClick={onNextClick} />
```

Comment:
Let's move this conditional out to a variable so we don't have to have a ternary inside the jsx